cmd/clusterctl/pkg/internal/test/fake_proxy.go

```diff
@@ -142,3 +144,27 @@ func (f *FakeProxy) WithProviderInventory(name string, providerType clusterctlv1
 
 	return f
 }
+
+// nameToProviderLabelName returns the ProviderLabelName for a provider.
+// Nb. this function mirrors util.NameToProviderLabelName that can't be used here to avoid circular dependencies
+func nameToProviderLabelName(name string, providerType clusterctlv1.ProviderType) string {
+	switch providerType {
+	case clusterctlv1.CoreProviderType:
+		return name
+	case clusterctlv1.BootstrapProviderType:
+		if name == "kubeadm-bootstrap" {
+			return "bootstrap-kubeadm"
+		}
+		return fmt.Sprintf("bootstrap-%s", name)
+	case clusterctlv1.ControlPlaneProviderType:
+		if name == "kubeadm-control-plane" {
```

Comment:
Same question as above

Reply:
changed approach; does not apply anymore